bitsail-common/src/main/resources/mysql-type-converter.yaml

```diff
@@ -0,0 +1,172 @@
+engine.type.to.bitsail.type.converter:
```

Comment:
@hk-lrzy thanks for your replay!
I also have this problem.

when I create a JdbcTypeInfoConverter via 'new JdbcTypeInfoConverter(StorageEngine.mysql.name())'
It will try to find 'mysql-type-converter.yaml' in resources.

How about put this yaml file in test resources not 'com.bytedance.bitsail.common.resources'

or Is there better way todo?

Reply:
@hk-lrzy I think It's ok with me. already done.